truenas/webui
# NAS-112617 / 21.10 / NAS-112617: Adding ix-checkbox component

- Added a new ix-checkbox component
- Added support for disabled states in new form components
- Refactored group-form
- Temporarily adds error display

Testing:
1. Check Credentials -> Local Groups and make sure that editing and adding groups works correctly.
2. Check that Localization Form in General Settings does not have regressions.
